src/ipc/ipc3/handler.c

```diff
@@ -603,6 +603,9 @@ static int ipc_pm_context_save(uint32_t header)
 
 	sa_exit(sof_get());
 
+	/* do platform specific suspending */
+	platform_suspend(sof_get());
```

Comment:
why do this here and not after line 636 as the last thing to be done?

Reply:
> why do this here and not after line 636 as the last thing to be done?

The logic here is do the platform specific suspending first and then the generic one.

Reply:
Usually generic followed platform suspend code, this is maybe easier to follow if we change to ```platform_suspend_save_state()```

Reply:
> Usually generic followed platform suspend code, this is maybe easier to follow if we change to `platform_suspend_save_state()`

'platform_context_save()'?